Makefile

```diff
@@ -44,7 +44,7 @@ all build:
 #   make check
 #   make check WHAT=pkg/build GOFLAGS=-v
 check:
-	TEST_KUBE=1 hack/test-go.sh $(WHAT) $(TESTS) $(TESTFLAGS)
+	TEST_KUBE=1 JUNIT=1 hack/test-go.sh $(WHAT) $(TESTS) $(TESTFLAGS)
```

Comment:
Before this merges, this will have to be removed as the default.  openshift-vagrant can come back and specifically request it.

I understand why its here now while you're actively working on it.